tests/gpflow/kernels/test_kernels.py

```diff
@@ -172,7 +172,11 @@ def test_periodic_bad_ard_period():
         gpflow.kernels.Periodic(base_kernel, period=[1.0, 1.0, 1.0])
 
 
-kernel_setups = [kernel() for kernel in gpflow.kernels.Stationary.__subclasses__()] + [
+kernel_setups = [
+    kernel()
+    for stationary in gpflow.kernels.Stationary.__subclasses__()
+    for kernel in stationary.__subclasses__()
```

Comment:
```suggestion
    for kernel in gpflow.ci_utils.subclasses(gpflow.kernels.Stationary)
```
You will have to add `import gpflow.ci_utils` at the top of the file.

(Sorry, I missed this case in #1340!)